packages/components/addon/components/hds/table/table-sortable.js

```diff
@@ -0,0 +1,31 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+
+export default class HdsTableTableSortableComponent extends Component {
+  a11yMessageId = 'table-a11y-message-' + guidFor(this);
+
+  get sort() {
+    return `${this.args.sortBy}:${this.args.sortOrder}`;
+  }
+
+  @action
+  setSortBy(column) {
+    if (this.args.sortBy === column) {
+      //invert the sort order
+      this.args.setSortOrder(this.args.sortOrder === 'asc' ? 'desc' : 'asc');
+    } else {
+      this.args.setSortBy(column);
+      this.args.setSortOrder('asc');
+    }
+
+    const currentColHeading = event.target;
+    const currColHeadingText = currentColHeading.textContent;
+
+    const liveRegion = document.querySelector(`#${this.a11yMessageId}`);
+    liveRegion.textContent = `Sorted by ${currColHeadingText} ${this.args.sortOrder}ending`;
+    setTimeout(function () {
```

Comment:
On a second look at the code I _think_ you should store the [return value](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#return_value) of the `setTimeout` function and use it to clear the setTimeout call, in case the user sorts (clicks) two columns in sequence, and the clearing of the live region done by the first `setTimeout` interferes with the setting of the same live region by the second sorting event

Reply:
Also, you may have missed this comment: https://github.com/hashicorp/design-system/pull/615/files/e752418fbf1c284b88c1c6f25b371878cab8ca90#r991382455

Reply:
Another alternative re. the `setTimeout` would be... remove it entirely :)
I am not sure why are you resetting its content after two seconds? any accessibility-related reason? because the table remains still sorted after that time, so the content is still valid, and in case the user changes sorting criteria, the live region content will be updated anyway, so... can you explain why the need to reset (something I may be missing).